addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -443,11 +444,27 @@ class MessageList extends Component {
             return false;
         }
         const loadMoreRect = loadMore.getBoundingClientRect();
-        const elRect = this.el.getBoundingClientRect();
+        const elRect = this.scrollableParent.getBoundingClientRect();
```

Comment:
The same fix should be done in a lot of other places in this file. Double check every time we use `this.el` in the context of the scroll position.

Reply:
@seb-odoo . I have also considered the changes as you have mentioned above regarding the ```o_Chatter_scrollPanel``` like below

```const elRect = this.props.order === 'asc' ? this.el.getBoundingClientRect() : this.el.closest('o_Chatter_scrollPanel').getBoundingClientRect();```

This will resolve all the issues using only single-line change.. but I thought it should be more dependant on any JS class. So I have tried it using the dynamic getClosestParentElement Method.

Whats your view on this change ?

Reply:
Honestly I think it's bad either way, so might as well use the simplest one... We do want to target either this or `o_Chatter_scrollPanel`, nothing else, so it should not be generic.

But ideally we should not have to do that of course... I'm still not sure if this approach is a good idea. Also `_onScroll` handler will still be broken due to being registered on a non-scrollable element in thos case.

Reply:
> Honestly I think it's bad either way, so might as well use the simplest one... We do want to target either this or `o_Chatter_scrollPanel`, nothing else, so it should not be generic.
> 
> But ideally we should not have to do that of course... I'm still not sure if this approach is a good idea. Also `_onScroll` handler will still be broken due to being registered on a non-scrollable element in this case.

@seb-odoo . Just check the new Implementation. Instead of using the onScroll method of the message list(some of the code is not required in our case.). We can achieve it using only calling the ```_loadMore``` method of the message list with minimal code implementation.